File: 2015/Saliendo_de_la_crisis/Crisis.c
#include <stdio.h>

int main()
{
  int nDatos;
  scanf("%d", &nDatos);
  while (nDatos != 0) {
    int n0;
    scanf("%d", &n0);
    int r = 1;
    int i;
    for (i = 0; i < nDatos - 1; i++) {
      int n1;
      scanf("%d", &n1);
      if (n0 >= n1)
        r = 0;
      n0 = n1;
    }
    if (r == 1)
      printf("SI\n");
    else
      printf("NO\n");
    scanf("%d", &nDatos);
  }
}

File: 2015/Saliendo_de_la_crisis/Crisis.java
import java.util.Scanner;

public class Crisis {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int nDatos = sc.nextInt();
        while (nDatos != 0) {
            int n0 = sc.nextInt();
            boolean r = true;
            for (int i = 0; i < nDatos - 1; i++) {
                int n1  = sc.nextInt();
                if (n0 >= n1)
                    r = false;
		n0 = n1;
            }
            if (r) {
                System.out.println("SI");
            } else
                System.out.println("NO");
            nDatos = sc.nextInt();
        }
        sc.close();
    }
}
